app/models/dialect.rb

```diff
@@ -1,3 +1,4 @@
 class Dialect < ApplicationRecord
   has_many :grammars
+  has_many :place_infos
```

Comment:
```suggestion
  has_many :place_infos, strict_loading: true
```

Might be good to try so you avoid N+1 queries! https://blog.saeloun.com/2020/02/25/rails-strict-loading-mode-to-fix-n-1.html

Reply:
Never mind that is only for rails 6.1!